Source/WebSocket/Websocketpp/websocketpp_websocket.cpp

```diff
@@ -809,7 +811,9 @@ HRESULT CALLBACK Internal_HCWebSocketSendBinaryMessageAsync(
     _In_ HCWebsocketHandle websocket,
     _In_reads_bytes_(payloadSize) const uint8_t* payloadBytes,
     _In_ uint32_t payloadSize,
-    _Inout_ XAsyncBlock* asyncBlock)
+    _Inout_ XAsyncBlock* asyncBlock,
+    _In_opt_ void* context
```

Comment:
Also shouldn't the implementation be updated so this actually gets passed back to the callback? here and elsewhere?

Reply:
no this context is for this function, it's not what the consumer registers with their own callbacks (that comes out in get event handlers)